### ISSUE TITLE
add support for model compositionality with keras

### ISSUE THREAD

**Issue by dustinvtran** [blei-lab/edward#17]
Seems like [keras](http://keras.io) is winning out of all the deep learning abstraction-based libraries. 

It would great to be compatible with [keras' compositionality](http://keras.io/models/) for adding neural network layers. This would be convenient for stacking NN layers following their Sequential and Graph Model classes, as options to parameterize the probability model and/or the variational model.

It looks like all we need to do is write a wrapper around their `evaluate()` method, and hack evaluation of the log-likelihood as the compiled model object's "loss".
* https://github.com/fchollet/keras/blob/master/keras/models.py
* http://blog.keras.io/keras-as-a-simplified-interface-to-tensorflow-tutorial.html
